foxy/distribution.yaml

```diff
@@ -1079,7 +1079,7 @@ repositories:
       tags:
         release: release/foxy/{package}/{version}
       url: https://github.com/flynneva/grbl_ros-release.git
-      version: 0.2.1-1
+      version: 0.0.14-1
```

Comment:
Typically version numbers go up (not down). Is this change correct?

Reply:
im switching from managing distro-specific branches to a single-source style. the old version reflects the distro-specific approach that was used before.

Reply:
If users have the previous version 0.2.1 installed they won't be able to update to 0.0.14 (since the version number is lower). They will need to uninstall and reinstall instead. Is that still desired / acceptable?

Reply:
no one is using this package since the previous released version does not even function really. so yeah thats fine.

Reply:
i would be shocked if a single person other than myself has tried out this grbl_ros package to be completely honest :smile: